notification/twilio/message.go

```diff
@@ -94,8 +95,10 @@ func (msg *Message) messageStatus(id string) *notification.MessageStatus {
 		status.State = notification.MessageStateDelivered
 	case MessageStatusSent, MessageStatusUndelivered:
 		status.State = notification.MessageStateSent
+	case MessageStatusAccepted, MessageStatusQueued:
+		status.State = notification.MessageStatePending
```

Comment:
We can drop this case -- any state from Twilio that isn't final should drop to `Sending` below
```suggestion
```